dbtmetabase/__init__.py

```diff
@@ -6,112 +6,165 @@
 from .metabase import MetabaseClient
 from .parsers.dbt_folder import DbtFolderReader
 from .parsers.dbt_manifest import DbtManifestReader
+from .models.config import MetabaseConfig, dbtConfig
 
-from typing import Iterable, List, Union
+from typing import Iterable, List, Union, Optional
 
 from ._version import version as __version__
 
 
-def export(
-    dbt_database: str,
-    metabase_database: str,
-    metabase_host: str,
-    metabase_user: str,
-    metabase_password: str,
-    dbt_manifest_path: str = "",
-    dbt_path: str = "",
-    dbt_docs_url: str = None,
-    metabase_use_http: bool = False,
-    metabase_verify: Union[str, bool] = True,
-    metabase_sync_skip: bool = False,
-    metabase_sync_timeout: int = None,
-    schema: str = "public",
-    schema_excludes: Iterable = None,
-    includes: Iterable = None,
-    excludes: Iterable = None,
+def models(
+    metabase_config: MetabaseConfig,
+    dbt_config: dbtConfig,
     include_tags: bool = True,
+    dbt_docs_url: Optional[str] = None,
 ):
     """Exports models from dbt to Metabase.
 
     Args:
-        dbt_database (str): Source database name.
-        metabase_database (str): Target Metabase database name. Database in Metabase is aliased.
-        metabase_host (str): Metabase hostname.
-        metabase_user (str): Metabase username.
-        metabase_password (str): Metabase password.
-        dbt_manifest_path (str, optional): Path to dbt project manifest.json [Primary]. Defaults to "".
-        dbt_path (str, optional): Path to dbt project. [Alternative]. Defaults to "".
+        metabase_config (str): Source database name.
+        dbt_config (str): Target Metabase database name. Database in Metabase is aliased.
+        include_tags (bool, optional): Append the dbt tags to the end of the table description. Defaults to True.
         dbt_docs_url (str, optional): URL to your dbt docs hosted catalog, a link will be appended to the model description (only works for manifest parsing). Defaults to None.
-        metabase_use_http (bool, optional): Use HTTP to connect to Metabase instead of the default HTTPS. Defaults to False.
-        metabase_verify (Union[str, bool], optional): Supply path to certificate or disable verification. Defaults to True.
-        metabase_sync_skip (bool, optional): Skip synchronizing Metabase database before export. Defaults to False.
-        metabase_sync_timeout (int, optional): Metabase synchronization timeout in seconds. Defaults to None.
-        schema (str, optional): Target schema name. Defaults to "public".
```

Comment:
For consistency, should all arguments specific to dbt be prefixed with `dbt_`? Per @z3z1ma in [this comment](https://github.com/gouline/dbt-metabase/pull/38#issuecomment-885781332) schema for ex applies to the schemas from dbt. 

-  `schema` -> `dbt_schema`
- `exclude_schema` -> `dbt_exclude_schema`
- `includes` -> `dbt_include_models`
- `excludes` -> `dbt_exclude_models`

Those names are longer but also convey more context about their purpose.

Reply:
With the addition of the new command, we are working in CLI improvements in this PR. So I'm totally open to the idea of making those specific flags more clear @erika-e. `schema` used to be used in the context of Metabase but with the introduction of the hash key of schema.table being used for Metabase references, it became unnecessary to filter the Metabase table and field lookups. Instead filtering the dbt project for a subset of models was sufficient to scope the execution. 

I wanted to avoid the breaking change flag on this one but if it's an improvement for potential users and existing users I think @goulline would agree.

Reply:
Don't worry about breaking changes, we already made _a lot_ of breaking changes.

Reply:
I could take on making the arguement name changes in a separate PR, since there's a lot going on with this one already. @gouline would you prefer that we make issues for things like this? I'm happy to make one if you like.

Reply:
I'd say yes for something large, but for renaming arguments feel free to just do it and reference this discussion thread in your PR description.